### PR TITLE
Fix typos in pkg/network

### PR DESCRIPTION
**Pull request by pravisankar**

### CODE REVIEW

**Comment by pravisankar** [comment]
@openshift/networking PTAL

File: pkg/network/apis/network/validation/validation.go [hunk 1]
```diff
@@ -36,9 +37,9 @@ func ValidateClusterNetwork(clusterNet *networkapi.ClusterNetwork) field.ErrorLi
 		} else {
 			maskLen, addrLen := clusterIPNet.Mask.Size()
 			if clusterNet.HostSubnetLength > uint32(addrLen-maskLen) {
-				allErrs = append(allErrs, field.Invalid(field.NewPath("hostSubnetLength"), clusterNet.HostSubnetLength, "subnet length is too large for clusterNetwork"))
```
Comment (rajatchopra) [inline pkg/network/apis/network/validation/validation.go]: Are you sure this should not be camelCase?

Comment (pravisankar) [reply pkg/network/apis/network/validation/validation.go]: Yes (refer https://github.com/openshift/origin/blob/master/pkg/network/apis/network/v1/types.go)
Even though camel case is the general norm, we incorrectly published the field name as 'hostsubnetlength', so this error should refer to the same name.
Anyway this field under ClusterNetwork is deprecated and the new HostSubnetLength under ClusterNetworkEntry is using camel case notation.

File: pkg/network/master/master.go [hunk 2]
```diff
@@ -94,7 +94,7 @@ func Start(networkConfig osconfigapi.MasterNetworkConfig, networkClient networkc
 			log.Infof("Created ClusterNetwork %s", common.ClusterNetworkToString(configCN))
 
 			if err = master.checkClusterNetworkAgainstClusterObjects(); err != nil {
-				log.Errorf("WARNING: cluster contains objects incompatible with new ClusterNetwork: %v", err)
+				log.Warningf("Cluster contains objects incompatible with new ClusterNetwork: %v", err)
```
Comment (danwinship) [inline pkg/network/master/master.go]: I had made this a log.Error intentionally to make it more likely to get noticed in the logs. (You can remove the "WARNING" if you want, but I think this should stay as log.Error.)

File: pkg/network/master/master.go [hunk 3]
```diff
@@ -105,7 +105,7 @@ func Start(networkConfig osconfigapi.MasterNetworkConfig, networkClient networkc
 				configCN.TypeMeta = existingCN.TypeMeta
 				configCN.ObjectMeta = existingCN.ObjectMeta
 				if err = master.checkClusterNetworkAgainstClusterObjects(); err != nil {
-					log.Errorf("ERROR: Attempting to modify cluster to excludes existing objects: %v", err)
+					log.Errorf("Attempting to modify cluster to excludes existing objects: %v", err)
```
Comment (danwinship) [inline pkg/network/master/master.go]: "excludes" should be "exclude" there...

File: pkg/network/node/networkpolicy.go [hunk 4]
```diff
@@ -148,7 +148,7 @@ func (np *networkPolicyPlugin) AddNetNamespace(netns *networkapi.NetNamespace) {
 	defer np.lock.Unlock()
 
 	if _, exists := np.namespaces[netns.NetID]; exists {
-		glog.Warning("Got AddNetNamespace for already-existing namespace %s (%d)", netns.NetName, netns.NetID)
+		glog.Warningf("Got AddNetNamespace for already-existing namespace %s (%d)", netns.NetName, netns.NetID)
```
Comment (danwinship) [inline pkg/network/node/networkpolicy.go]: Isn't govet supposed to catch those?
...
OK, apparently, no, it looks for Logf, Errorf, and Fatalf, but not Warningf. We need to pass an argument. I'll file a bug about that.

**Comment by pravisankar** [comment]
@danwinship updated, ptal

**Comment by imcsk8** [comment]
LGTM

**Pull request merged**